server/drizzle/0033_changes3.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE "link_embed_experimental" RENAME COLUMN "provider" TO "provider_name";--> statement-breakpoint
```

Comment:
delete all your migrations, reset your db, do one new migration to consolidate them all.